### ISSUE TITLE
Incorrect usage of e.directed

### ISSUE THREAD

**Issue by tarunkle** [hadim/pygraphml#30]
https://github.com/hadim/pygraphml/blob/8c719ede0f5c6e22367b499d8dfaf54852bb40b2/pygraphml/graphml_parser.py#L67

Hello,
The directed attribute of e is not a function it is a boolean attribute.
Here is the error I am getting:
  File "/users/trunkle/anaconda2/lib/python2.7/site-packages/pygraphml/graphml_parser.py", line 67, in write
    if e.directed() != graph.directed:
TypeError: 'bool' object is not callable

Suggest removing the parens () from directed on two lines, line 67 and line 68.
Works for me.
Thanks,
Tim.

**Comment by hadim** [comment]
Thank you for the bug report.

Can you submit a PR to fix this?

**Comment by tarunkle** [comment]
Hello,
Happy to help. Assume PR is pull request? Can you provide instructions or link to instructions?
Thanks,
Tim.

**Comment by hadim** [comment]
Yes, PR means pull request. Here is a tutorial: https://help.github.com/articles/about-pull-requests/
